api/Window.json

```diff
@@ -509,6 +509,54 @@
           }
         }
       },
+      "find": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Window/find",
+          "support": {
+            "chrome": {
+              "version_added": true
```

Comment:
The [commit for this](https://chromium.googlesource.com/chromium/src/+/f813c3fae96d7b31f71d93595ff71363700952a1) predates version 1. Therefore `chrome` is `, `chrome_android` is 18, and `webview_android` is `true`.